### ISSUE TITLE
Very high Accuracy Loss in custom dataset

### ISSUE THREAD

**Issue by alexandoikon13** [alankbi/detecto#92]
I used all the instructions given for training Faster R-CNN with Detecto on a custom dataset, had no issues there. I was able to successfully use all functions of Detecto. However, after training on a custom dataset with a different number of epochs, the accuracy loss was extremely high all the time (eg. 0.66, 0.95, 1.01 etc.), making no sense compared to visualizing model's performance on making predictions. Also, the same custom dataset was used in other model which returned "normal" loss values.

Any idea why is that or if there is a bug on the function?

**Comment by alankbi** [comment]
Not sure - it's hard to say anything without having more information about the scenario. However, you said that the predictions are working fine?

**Comment by alexandoikon13** [comment]
Yes, the predictions are working fine. It's only the accuracy loss graph not making sense.

**Comment by alankbi** [comment]
In that case it's possible it's not a bug with the loss values - feel free to take a look at the [torchvision documentation](https://pytorch.org/vision/stable/models.html#object-detection-instance-segmentation-and-person-keypoint-detection) to learn about what the loss values represent.
